### PR TITLE
Run rspec in CI

### LINKED ISSUE #35: Create a Github Actions pipeline that runs tests
Task broken out of #33

### PR DESCRIPTION
**Pull request by hartsick**
closes #35

### CODE REVIEW

File: .github/workflows/rspec.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: ./.github/actions/setup-project
+
+      - uses: ./.github/actions/setup-database
+
+      - name: Run rspec
+        run: bundle exec rspec
```
Comment (rahearn) [inline .github/workflows/rspec.yml:31]: [suggestion] passing the `DATABASE_URL` environment variable will let the tests see the db

**Comment by hartsick** [comment]
All set! Intending to squash & merge this onto main when time comes

File: .github/workflows/rspec.yml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+      - uses: actions/checkout@v2
+
+      - uses: ./.github/actions/setup-project
+
+      - id: database-setup
+        uses: ./.github/actions/setup-database
+
+      - name: Run rspec
+        env:
+          DATABASE_URL: ${{ steps.database-setup.outputs.database_url }}
```
Comment (rahearn) [inline .github/workflows/rspec.yml:33]: [praise] I wish this were a _little_ more succinct, but I think you've made it as short and easy as possible. Love it.

**Review by rahearn — APPROVED** [review]
Love it

**Comment by rahearn** [comment]
Squash and merge complete so I can use that awesome output stuff

**Pull request merged**